scripts/r_rnaseq_packages.R

```diff
@@ -0,0 +1,7 @@
+# R packages for RNA sequence workshop part 1 
+
+install.packages(c("ggplot2", "pheatmap", "RColorBrewer", "PoiClaClu",
+                   "patchwork", "BiocManager", "tidyr", "GGally"), dependencies=TRUE, 
+                 repos='http://cran.rstudio.com/')
+BiocManager::install(c("airway", "DESeq2", "vsn", "biomaRt",
+                       "AnnotationHub", "SummarizedExperiment"))
```

Comment:
1. Base R packages are installed for every R-hub... maybe we should move ggplot2 and tydr to the `requirements.txt` file so conda installs that for every R class. In fact in your case, if anything else can be installed from `conda-forge` you may want to move it there .... dependency management may be better.

2. For now  the best is to move this file to the hub specific folder i.e, `equirements/classes/deseq_workshop_1`. The contents of that folder are already available to the the docker host under `/home/$NB_USER/tmp/` - see https://github.com/brown-ccv/jupyterhub-docker-images/blob/8771e9819386e8e007102f60e5f4e0f72937ee74/Dockerfile#L108
3. Rename file to something more generic i.e `packages.R`. 
4.  Since we share 1 Dockerfile for all projects. We want to make sure, when someone generates a new class, the build wil work. So we also need to modify the `dev/templates` folder to include a "template" of this script - the template could just be your contents commented out as an example. But the post important part is that when in the docker file you call `RUN Rscript packages.R` the line works if this file is empty

Reply:
I think that using conda for R packages makes it worse. We want to have as much of the R packages installed from either CRAN or Bioconductor etc

Reply:
we've come a long way ... I though I was the only one that didn't like conda 😂 

In that case, you can decide and installed them as preferred.

But... Do you think that we should be generally not installing R packages from Conda? If so we could move all R packages out of the `requirements.txt` file (maybe in future PR). I don't install Julia packages via `conda` either